content/security/for-admins/hardened-desktop/settings-management/admin-console-configure.md

```diff
@@ -0,0 +1,68 @@
+---
+description: How to configure Settings Management for Docker Desktop using the Docker Admin Console
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure with the Docker Admin Console
+---
+
+{{% restricted title="Early Access" %}}
+
+Settings Management in Docker Admin Console is in Early Access and is available to Docker Business customers only.
+
+{{% /restricted %}}
+
+This page contains information for administrators on how to configure Settings Management with the Docker Admin Console to specify and lock configuration parameters to create a standardized Docker Desktop environment across your Docker company or organization.
+
+## Prerequisites
+
+- [Download and install Docker Desktop 4.35.0 or later](/desktop/release-notes.md).
+- As an administrator, you need to [enforce
```

Comment:
Wondering if we need to call that out as a requirement. While we need the users to login with DB subscription, maybe we suggest "Enforce Signin" as a "recommendation" to make policy compliance more effective?